plugins/funder.c

```diff
@@ -1280,7 +1280,7 @@ int main(int argc, char **argv)
 				  " being advertised",
 				  bool_option,
 				  &current_policy->leases_only),
-		    plugin_option("lease-fee-base-msat",
+		    plugin_option("lease-fee-base-sat",
```

Comment:
Good catch!

But you need to put back the old name, under 'if (deprecated_apis)' (note in CHangelog-Experimental that it's deprecated and will be removed next release).

(We usually deprecate over 2 releases, but experimental stuff can move faster)